Permanent/Modules/EditMetadata/ViewModels/MetadataEditFileNamesViewModel.swift

```diff
@@ -25,3 +25,10 @@ class MetadataEditFileNamesViewModel: ObservableObject {
         }
     }
 }
+
+enum FilenameAction {
```

Comment:
Are you using this enum anywhere? Is there a reason we have none here?

Reply:
I don't think we need it anymore.